src/screens/ProfileSheet.tsx

```diff
@@ -104,7 +105,10 @@ export default function ProfileSheet() {
         <AccentColorProvider color={accentColor}>
           <Box background="body">
             <Box style={wrapperStyle}>
-              {!isPreview && (!isAddressSuccess || !hasListFetched) ? (
+              {!isPreview &&
+              (!isAddressSuccess ||
+                !hasListFetched ||
+                !hasFirstTxTimestampFetched) ? (
```

Comment:
May make sense to extract that boolean expression to a variable and make it somewhat descriptive like `shouldShowFoo`